kubernetes/07-mongodb-statefulset.yaml

```
@@ -1,35 +1,35 @@
-              mountPath: /data/db
-          resources:
-            requests:
-              memory: "256Mi"
-              cpu: "250m"
-            limits:
-              memory: "512Mi"
-              cpu: "500m"
-      volumes:
-        - name: mongodb-storage
-          persistentVolumeClaim:
+apiVersion: apps/v1
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)